Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/.gitignore b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1699456120196)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/.gitignore	(date 1699456120196)
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/modules.xml b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1699456120227)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/modules.xml	(date 1699456120227)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/DjangoModelsRelationsExercise.iml" filepath="$PROJECT_DIR$/.idea/DjangoModelsRelationsExercise.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/dataSources.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/dataSources.xml b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/dataSources.xml
new file mode 100644
--- /dev/null	(date 1699456120180)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/dataSources.xml	(date 1699456120180)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
+    <data-source source="LOCAL" name="postgres@localhost" uuid="964b3798-452a-4c92-91f0-d628d70c0b81">
+      <driver-ref>postgresql</driver-ref>
+      <synchronize>true</synchronize>
+      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
+      <jdbc-url>jdbc:postgresql://localhost:5432/postgres</jdbc-url>
+      <working-dir>$ProjectFileDir$</working-dir>
+    </data-source>
+  </component>
+</project>
\ No newline at end of file
Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/DjangoModelsRelationsExercise.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/DjangoModelsRelationsExercise.iml b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/DjangoModelsRelationsExercise.iml
new file mode 100644
--- /dev/null	(date 1699456120227)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/DjangoModelsRelationsExercise.iml	(date 1699456120227)
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="FacetManager">
+    <facet type="django" name="Django">
+      <configuration>
+        <option name="rootFolder" value="$MODULE_DIR$" />
+        <option name="settingsModule" value="orm_skeleton/settings.py" />
+        <option name="manageScript" value="$MODULE_DIR$/manage.py" />
+        <option name="environment" value="&lt;map/&gt;" />
+        <option name="doNotUseTestRunner" value="false" />
+        <option name="trackFilePattern" value="migrations" />
+      </configuration>
+    </facet>
+  </component>
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+  <component name="PyDocumentationSettings">
+    <option name="format" value="PLAIN" />
+    <option name="myDocStringFormat" value="Plain" />
+  </component>
+  <component name="TemplatesService">
+    <option name="TEMPLATE_CONFIGURATION" value="Django" />
+  </component>
+</module>
\ No newline at end of file
Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/misc.xml b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1699456120149)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/misc.xml	(date 1699456120149)
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.11 (DjangoModelsRelationsExercise)" project-jdk-type="Python SDK" />
+</project>
\ No newline at end of file
Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/Project_Default.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/Project_Default.xml b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/Project_Default.xml
new file mode 100644
--- /dev/null	(date 1699456120180)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/Project_Default.xml	(date 1699456120180)
@@ -0,0 +1,22 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="PyPep8NamingInspection" enabled="true" level="WEAK WARNING" enabled_by_default="true">
+      <option name="ignoredErrors">
+        <list>
+          <option value="N801" />
+          <option value="N802" />
+        </list>
+      </option>
+    </inspection_tool>
+    <inspection_tool class="PyStatementEffectInspection" enabled="false" level="WARNING" enabled_by_default="false" />
+    <inspection_tool class="PyUnresolvedReferencesInspection" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoredIdentifiers">
+        <list>
+          <option value="Python_Advanced_September.Python_OOP_October.Week_1.Exercise_2.project" />
+          <option value="django.contrib.admin.mysite" />
+        </list>
+      </option>
+    </inspection_tool>
+  </profile>
+</component>
\ No newline at end of file
Index: Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/profiles_settings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/profiles_settings.xml b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/profiles_settings.xml
new file mode 100644
--- /dev/null	(date 1699456120149)
+++ b/Python DB/PythonORM/DjangoModelsRelationsExercise/.idea/inspectionProfiles/profiles_settings.xml	(date 1699456120149)
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
\ No newline at end of file
